teachopencadd/talktorials/T032_compound_activity_proteochemometrics/talktorial.ipynb

```diff
@@ -0,0 +1,1923 @@
+{
```

Comment:
<p>an effect of the observed results &gt; an effect <strong>on</strong> the observed results</p><p>The LOTO split is harder <strong>in validation</strong> than the random split &gt; The LOTO split is harder <strong>to validate (?)</strong> than the random split <strong>(or what is meant by "harder in validation"?)</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/278/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='93'/>

Reply:
<p>I changed it to : The LOTO split is a more challenging form of validation </p><p> </p>